action.yml

```diff
@@ -19,6 +19,10 @@ inputs:
   awsAccount:
     description: 'AWS Account ID'
     required: true
+  useLatestTemplate:
```

Comment:
I prefer to replace `template` with `manifest` (suggestion: `latestIaCManifest`).

Reply:
maybe just `useLatestManifest` ? I would be careful using the term IaC, as that would indicate that it is being applied as infrastructure, which is not the case.

Reply:
nice